src/CPUSolver.cpp

```diff
@@ -185,6 +185,28 @@ void CPUSolver::setNumThreads(int num_threads) {
 
 
 /**
+ * @brief Initializes the FSR volumes and Materials array.
+ * @details This method assigns each FSR a unique, monotonically increasing
+ *          ID, sets the Material for each FSR, and assigns a volume based on
+ *          the cumulative length of all of the segments inside the FSR.
+ */
+void CPUSolver::initializeFSRs() {
```

Comment:
Correct, I merged with your `fsr-volume-getter` branch so PR #134 should be merged first. If more of your PRs are merged into `develop`, I'll continue to update this PR.